File: BOJ_2460/BOJ_2460.py
import sys
ANS, PLACE_HOLDER = 0, 0
for _ in range(10):
    N, M = map(int, sys.stdin.readline().split())
    ANS -= N
    ANS += M
    if(ANS>PLACE_HOLDER):PLACE_HOLDER = ANS
print(PLACE_HOLDER)

File: BOJ_2460/BOJ_2460.cs
using System;
using System.Linq;
using System.Text;
namespace asharien{
	class PROGRAM{
		static void Main(){
			int ANS = 0, HOLDER = 0;
			for(int i=0; i<10; i++){
				int[] X = Console.ReadLine().Split().Select(int.Parse).ToArray();
				ANS -= X[0];
				ANS += X[1];
				if(ANS>HOLDER) HOLDER = ANS;
			}
			Console.WriteLine(HOLDER);

		}
	}
}
